examples/cross-contract-calls/README.md

```diff
@@ -1,5 +1,58 @@
-# Cross contract
+# Cross Contract Call
 
-Example of using cross-contract functions with promises.
+Example implementation of a cross-contract call using [near-sdk-rs].
 
-TBD
+[near-sdk-rs]: https://github.com/near/near-sdk-rs
+
+NOTES:
+
+- This example demonstrates how to call another contract from within a contract.
+- Ensure that the called contract is deployed and accessible.
+
+## Project Structure
+
+- `.cargo`: Configuration files for Cargo, the Rust package manager.
+- `high-level/src`: Source code for the high-level cross-contract call example.
+- `low-level/src`: Source code for the low-level cross-contract call example.
+- `res`: Compiled contract binaries.
+- `tests`: Integration tests for the cross-contract call examples.
+
+## Building
+
+To build run:
+
+```bash
+./build.sh
+```
+
+## Testing
+
+To test run:
+
+```bash
+cargo test --package cross-contract-call -- --nocapture
+```
+
+### Usage
+
+```markdown
```

Comment:
What is the benefit of having this block in ```markdown?

Reply:
There is no benefit other than aesthetic honestly, I can change it if you want! Just let me know @frol